packages/subgraph/CHANGELOG.md

```diff
@@ -0,0 +1,258 @@
+# Changelog
+
+All notable changes to the Subgraph will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.7.0] - 2024-02-05
```

Comment:
GDA features are from 1.7.0 right?

Reply:
yup

Reply:
shouldn't we then write a few texts about it?

Reply:
forgot to hit save after writing that part haha 🥲